src/main/cpp/Robot.cpp

```diff
@@ -27,25 +31,29 @@ void Robot::TeleopInit() {
 
 void Robot::TeleopPeriodic() {
 
-  double dx = ljoy.GetRawAxis(1);
-  double dy = ljoy.GetRawAxis(0);
-  double dtheta = rjoy.GetX();
-  dx = abs(dx) < 0.1 ? 0.0: dx; 
-  dy = abs(dy) < 0.1 ? 0.0: dy;
-  dtheta = abs(dtheta) < 0.05 ? 0.0: dtheta;
-
-  joy_val_to_mps(dx);
-  joy_val_to_mps(dy);
-  joy_rot_to_rps(dtheta);
-
+  //get joystick input
+  double vx = ljoy.GetRawAxis(1);
```

Comment:
why is this called "raw" and what does it do?

Reply:
It's called raw because for this function you pass in the "raw" axis number instead of calling the `getX()` and `getY()` axis getter functions. Mentioned this in an earlier comment, but since the function redundancy is confusing, next time I'm in shop I'll change this to be `getX()` or `getY()`, whichever one it is.